src/seq/nucleotide.jl

```diff
@@ -103,6 +103,24 @@ function isGC(nt::DNANucleotide)
 end
 
 """
+    isPurine(nt::Nucleotide)
+
+Test if nucleotide is surely a Purine.
+"""
+function isPurine(nt::Nucleotide)
+    return (compatbits(nt) | 0b0101) == 0b0101
```

Comment:
Can this be done with bits operations another way, or do we have to resort to a `nt == DNA_A || nt == DNA_G || nt == DNA_R`, and create seperate method defs out for DNANucleotide and RNANucleotide?